src/style.rs

```diff
@@ -529,4 +529,165 @@ mod tests {
             assert_eq!(input.resolve(parent), expected);
         }
     }
+
+    mod test_flex_direction {
+        use crate::style::*;
+
+        #[test]
+        fn flex_direction_is_row() {
+            assert_eq!(FlexDirection::Row.is_row(), true);
+            assert_eq!(FlexDirection::RowReverse.is_row(), true);
+            assert_eq!(FlexDirection::Column.is_row(), false);
+            assert_eq!(FlexDirection::ColumnReverse.is_row(), false);
+        }
+
+        #[test]
+        fn flex_direction_is_column() {
+            assert_eq!(FlexDirection::Row.is_column(), false);
+            assert_eq!(FlexDirection::RowReverse.is_column(), false);
+            assert_eq!(FlexDirection::Column.is_column(), true);
+            assert_eq!(FlexDirection::ColumnReverse.is_column(), true);
+        }
+
+        #[test]
+        fn flex_direction_is_reverse() {
+            assert_eq!(FlexDirection::Row.is_reverse(), false);
+            assert_eq!(FlexDirection::RowReverse.is_reverse(), true);
+            assert_eq!(FlexDirection::Column.is_reverse(), false);
+            assert_eq!(FlexDirection::ColumnReverse.is_reverse(), true);
+        }
+    }
+
+    mod test_flexbox_layout {
+        use crate::style::*;
+
+        fn size_from_points(width: f32, height: f32) -> Size<Dimension> {
```

Comment:
I kind of want these to be methods on Size / Rect / FlexboxLayout. Thoughts?

Reply:
Yeah I think that makes sense :)

Reply:
I think that it works great for the Size and Rect, they're nice shortcurts, but I'm not sure if it's useful for FlexboxLayout.
It seems a bit arbitrary to have only `from_align_self` and `from_align_items` and even then, I'm not sure if it's a common use case. Would love to have some shorthand for initializing a default FlexboxLayout though. 
So I would prefer any of those two options for FlexboxLayout:
- Implement `from_align_self`, `from_align_items` and every other `from_x`
- Don't implement any of them (I'm partial to this)

What do you guys think?

Reply:
I vote for adding these convenience methods to `Size` and `Rect`, then we can skip any changes to `FlexboxLayout` for now then revisit later.

Reply:
Great! I've added them and updated the tests. Let me know if there's anything I can improve.